src/App.jsx

```diff
@@ -57,30 +58,39 @@ function App() {
             </header>
             <Switch>
               <Route exact path="/">
-                <Redirect to="/condition" />
+                <Redirect to="/login" />
               </Route>
               <Route path="/login">
                 <Login />
               </Route>
-              <PrivateRoute exact path="/condition">
+              <PrivateRoute exact path="/:id">
```

Comment:
서버와 동일하게 :creator로 통일하시는 것이 어떤가요? 

```suggestion
              <PrivateRoute exact path="/:creator">
```

Reply:
/:id(=creator) 로 진행할 경우, 서버단의 /:id(=게시글oid)와 혼동될 가능성이 높아보입니다.

Reply:
좋습니다. 지금 유저 아이디를 받아오는 부분이 크게 아래 2가지 인데요,
1) url 상에 param로 들어오는 부분
2) 컴포넌트에서 selector로 받아오는 id (accessLevel별 처리 및 상세보기 페이지로 연결해주는 url 생성에 필요)
1번 같은 경우 제안해주신 대로 처리하는 것이 통일성이 있어 좋겠네요.
다만 2번 같은 경우에 store의 state명이 user로 잡혀있어서 user.creator처럼 접근하게 될 것 같아서 이전 db 스키마 관련해서 어색한 속성명 이슈가 발생하는 것 같아요. 이 부분 의견 부탁드립니다!

Reply:
state의 user.id와 경로의 :creator가 항상 동일하지는 않아서 반드시 같은 이름을 갖지 않아도 된다고 생각합니다. 
같은 타입의 변수라는 걸 나타내려면 :creatorId로 하거나,
아니면 통일감을 줘서 :userId로 하는 것도 좋을 것 같습니다.

Reply:
그렇군요. creator 그대로 유지하되, 셀렉터에서 데이터를 잡아올 때는 구조분해할당을 하지 않고 user.id 형태 그대로 사용하면 혼란이 방지될 것 같아요.
서버측에서도 url은 creator로, 요청을 보낸 측은 userId로 읽어오고 있으니 유사하게 user.id로 사용하면 괜찮을 것 같습니다.

Reply:
서버에서 id로 내려준 정보는 클라이언트에서 user.id (스토어 접근시) or userId (스토어 외부에서 접근 시), url 상의 정보는 creator로 통일하는 작업중입니다. 이와 관련해서 서버에 로그인 요청시 내려주는 정보도 userId로 주면 좋을 것 같습니다!

Reply:
서버에 적용하여 PR올렸습니다!

Reply:
빠른 반영 감사합니다!